lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -1321,15 +1321,8 @@ void LCIStrategicPlugin::plan_maneuvers_callback(
   }
 
   // get the lanelet that is on the route in case overlapping ones found
-  for (auto llt : current_lanelets)
-  {
-    auto route = wm_->getRoute()->shortestPath();
-    if (std::find(route.begin(), route.end(), llt) != route.end())
-    {
-      current_lanelet = llt;
-      break;
-    }
-  }
+  auto llt_on_route_optional = wm_->getLaneletOnShortestPath(current_lanelets);
+  current_lanelet = llt_on_route_optional ? llt_on_route_optional.value() : current_lanelets[0];
```

Comment:
Can you remind me what current_lanelets[0] here is?

Reply:
It is supposed to just pick arbitrary lanelet if no lanelet was found on the shortest path as a last resort